src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -150,63 +146,18 @@ export const monoPlatform: Platform = {
-  const loaderResponse = loadBootResource !== undefined ?
-    loadBootResource('manifest', 'blazor.boot.json', '_framework/blazor.boot.json', '') :
-    defaultLoadBlazorBootJson('_framework/blazor.boot.json');
-
-  let bootConfigResponse: Response;
-
-  if (!loaderResponse) {
-    bootConfigResponse = await defaultLoadBlazorBootJson('_framework/blazor.boot.json');
-  } else if (typeof loaderResponse === 'string') {
-    bootConfigResponse = await defaultLoadBlazorBootJson(loaderResponse);
-  } else {
-    bootConfigResponse = await loaderResponse;
-  }
-
-  // While we can expect an ASP.NET Core hosted application to include the environment, other
-  // hosts may not. Assume 'Production' in the absence of any specified value.
-  applicationEnvironment = environment || bootConfigResponse.headers.get('Blazor-Environment') || 'Production';
-  const bootConfig: BootJsonData = await bootConfigResponse.json();
-  bootConfig.modifiableAssemblies = bootConfigResponse.headers.get('DOTNET-MODIFIABLE-ASSEMBLIES');
-  bootConfig.aspnetCoreBrowserTools = bootConfigResponse.headers.get('ASPNETCORE-BROWSER-TOOLS');
-
-  return bootConfig;
-
-  function defaultLoadBlazorBootJson(url: string): Promise<Response> {
-    return fetch(url, {
-      method: 'GET',
-      credentials: 'include',
-      cache: 'no-cache',
-    });
-  }
-}
-
 async function importDotnetJs(startOptions: Partial<WebAssemblyStartOptions>): Promise<ModuleAPI> {
   const browserSupportsNativeWebAssembly = typeof WebAssembly !== 'undefined' && WebAssembly.validate;
   if (!browserSupportsNativeWebAssembly) {
     throw new Error('This browser does not support WebAssembly.');
   }
 
-  const bootConfig = await loadBootConfigAsync(startOptions.loadBootResource, startOptions.environment);
-
-  // The dotnet.*.js file has a version or hash in its name as a form of cache-busting. This is needed
-  // because it's the only part of the loading process that can't use cache:'no-cache' (because it's
-  // not a 'fetch') and isn't controllable by the developer (so they can't put in their own cache-busting
-  // querystring). So, to find out the exact URL we have to search the boot manifest.
-  const dotnetJsResourceName = Object
-    .keys(bootConfig.resources.runtime)
-    .filter(n => n.startsWith('dotnet.') && n.endsWith('.js'))[0];
-  const dotnetJsContentHash = bootConfig.resources.runtime[dotnetJsResourceName];
-  let src = `_framework/${dotnetJsResourceName}`;
+  let src = '_framework/dotnet.js';
 
   // Allow overriding the URI from which the dotnet.*.js file is loaded
   if (startOptions.loadBootResource) {
     const resourceType: WebAssemblyBootResourceType = 'dotnetjs';
-    const customSrc = startOptions.loadBootResource(resourceType, dotnetJsResourceName, src, dotnetJsContentHash);
+    const customSrc = startOptions.loadBootResource(resourceType, 'dotnet.js', src, '');
```

Comment:
This is probably a breaking change, because we don't have a hash of dotnet.js anymore